app/model/command/DeletePillarCommand.scala

```diff
@@ -0,0 +1,19 @@
+package model.command
+
+import com.gu.tagmanagement.{PillarEvent, PillarEventType}
+import repositories._
+import services.{Contexts, KinesisStreams}
+
+import scala.concurrent.Future
+
+
+case class DeletePillarCommand(id: Long) extends Command {
+  override type T = Long
+
+  override def process()(implicit username: Option[String] = None): Future[Option[Long]] = Future {
+    KinesisStreams.pillarUpdateStream.publishUpdate(id.toString, PillarEvent(PillarEventType.Delete, id, None))
```

Comment:
Do we want to free up the path in path manager here?

Will this result in content being modified in CAPI?

Reply:
pillars are not stored on the content. As requests come in capi adds the pillar to the content response based on the section.